lib/ibm/cloud/sdk/power_iaas.rb

```diff
@@ -15,6 +16,7 @@ def initialize(region, guid, token, crn, tenant)
           @region = region
           @token  = token
           @tenant = tenant
+          RestClient.log = Logging.logger
```

Comment:
I don't think Logging.logger exists unless you define it as `def self.logger` up in the Logging module.

Reply:
I'm actually surprised tests are passing

Reply:
I think you want to `include Logging` then you can just use `logger`